### PR TITLE
Smartling translations are completed

### PR DESCRIPTION
**Pull request by usethecodeluke**
Please review translation and merge this pull request

### CODE REVIEW

File: i18n/ja-JP/common.json [hunk 1]
```diff
@@ -1,115 +1,115 @@
...
   },
-  "account.product.header": {
-    "translation": "__NOT_TRANSLATED__"
+  "account.product.header" : {
+    "translation" : "アカウント製品"
   },
-  "account.product.navigator": {
-    "translation": "__NOT_TRANSLATED__"
+  "account.product.navigator" : {
+    "translation" : "Navigator"
```
Comment (abrande) [inline i18n/ja-JP/common.json]: Is this suppose to be translated?

File: i18n/ja-JP/validation.json [hunk 2]
```diff
@@ -1,34 +1,34 @@
...
+    "translate_paths" : {
+      "instruction" : "*/instruction",
+      "key" : "{*}/translation",
+      "path" : "*/translation"
     }
   },
-  "dropdown.role.mustBeSelected": {
-    "translation": "__NOT_TRANSLATED__"
+  "dropdown.role.mustBeSelected" : {
+    "translation" : "__NOT_TRANSLATED__"
```
Comment (abrande) [inline i18n/ja-JP/validation.json]: Note: This line has been deleted in a PR that is not accepted yet.

File: i18n/ja-JP/common.json (hunk 1 above)
Comment (usethecodeluke) [reply i18n/ja-JP/common.json]: no.  Weve been told product names do not get translated as they are brand marks.

File: i18n/ja-JP/validation.json (hunk 2 above)
Comment (usethecodeluke) [reply i18n/ja-JP/validation.json]: ok

**Pull request merged**